src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java

```diff
@@ -72,4 +82,25 @@ public static String formatCommandPrefix(TextChannel channel, String usage) {
         String prefix = String.valueOf(get(channel));
         return usage.replaceAll("\\{%\\}", prefix);
     }
+
+    public static AudioItem resolveItem(Player player, String input) throws IllegalArgumentException, YoutubeAccessException {
+        Optional<AudioItem> item = Optional.empty();
+        boolean failed = false;
+        for (int i = 0; i <= 2; i++) {
+            try {
+                item = Optional.ofNullable(player.resolve(input));
+                failed = false;
```

Comment:
Not even a point setting failed here as it breaks. This just wastes a bit of operation time, macroseconds yes but still operation time.